modules/cnvkit/batch/main.nf

```diff
@@ -18,28 +18,51 @@ process CNVKIT_BATCH {
     tuple val(meta), path("*.cnn"), emit: cnn, optional: true
     tuple val(meta), path("*.cnr"), emit: cnr, optional: true
     tuple val(meta), path("*.cns"), emit: cns, optional: true
+    tuple val(meta), path("*.pdf"), emit: pdf, optional: true
+    tuple val(meta), path("*.png"), emit: png, optional: true
     path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def normal_args = normal ? "--normal $normal" : ""
-    def fasta_args = fasta ? "--fasta $fasta" : ""
+    // execute samtools only when cram files are input
```

Comment:
Suggestion for this section: For readability, add some blank lines after groups of parameters that belong together